docs/developer/unified-repositories.rst

```diff
@@ -0,0 +1,42 @@
+Unified repository sourcing
+===========================
+
+The Unified feature is used automatically when Anaconda is loading installation repositories for the first time. This feature is used only if package (DNF) installation is active and the remote repository source is configured correctly. It was implemented to simplify the installation experience on systems where multiple repository sources are required to make a successful installation e.g. RHEL, CentOS Stream.
```

Comment:
DNF -> RPM
The repositories are contain RPM packages, so that's why I find "DNF" a bit confusing.